modules/@apostrophecms/schema/ui/apos/components/AposSearchList.vue

```diff
@@ -32,16 +33,37 @@ export default {
       default() {
         return [];
       }
+    },
+    disabledTooltip: {
+      type: String,
+      required: false
     }
   },
   emits: [ 'select' ],
   methods: {
-    select(item) {
+    select(item, $event) {
+      if (item.disabled) {
+        $event.stopPropagation();
```

Comment:
It's right to do this, but it's not enough to prevent dismissal of the list because that's done via a focusOut handler another level up, and we can't completely refuse all interaction on these because that would prevent them from having a tooltip, and thus the self-documenting function that is the entire reason we show them even though they are unpublished. Fortunately it's unlikely folks will click on them much once they see the tooltip, and the disabled appearance, and the fact that they don't get selected. So calling it an improvement, at the least.